### PR TITLE
Network drives

### PR DESCRIPTION
**Pull request by mrrys00**
This branch include tutorial how to make network drive on your local network

### CODE REVIEW

File: Create_local_network_drive/create_local_network_drive.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+
+## Wstęp
+Stworzenie i mapowanie dysku sieciowego w sieci lokalnej to bardzo prosty, wygodny i darmowy sposób na stworzenie domowego serwera plików (np. zdjęć, filmów, dokumentów, ...), do których możemy mieć dostęp z dowolnego innego komputera podłączonego do sieci domowej.
+
+## __Windows__
+
+## 1. Ustawianie statycznego IP routerze dla komputera z dyskiem sieciowym
+__UWAGA:__ Pierwsze 3 wartości wybranego adresu IP __muszą__ być zgodne z pierwszymi trzema polami _Bramy domyślnej_ / _Serwera DHCP_; czwarta wartość musi należeć do przedziału od _2 do 254_ (zalecane: _100 - 200_)
+1. Sprawdź _adres MAC_ swojego urządzenia (karty sieciowej za pomocą której łączysz się z Internetem): 
+    * Otwórz __Wiersz poleceń__ (__Command prompt__) i wpisz polecenie `ipconfig /all`, znajdź interfejs za pomocą którego komputer łączy się z internetem i pole _Physical Address_ <br/>
```
Comment (piotrekvs) [inline Create_local_network_drive/create_local_network_drive.md]: znajdź interfejs za pomocą (przecinek) którego

File: Create_local_network_drive/create_local_network_drive.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+---
+
+# Tworzenie i mapowanie  w sieci lokalnej
+
+## Wstęp
+Stworzenie i mapowanie dysku sieciowego w sieci lokalnej to bardzo prosty, wygodny i darmowy sposób na stworzenie domowego serwera plików (np. zdjęć, filmów, dokumentów, ...), do których możemy mieć dostęp z dowolnego innego komputera podłączonego do sieci domowej.
+
+## __Windows__
+
+## 1. Ustawianie statycznego IP routerze dla komputera z dyskiem sieciowym
```
Comment (piotrekvs) [inline Create_local_network_drive/create_local_network_drive.md]: IP (W) routerze

File: Create_local_network_drive/create_local_network_drive.md [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+![ArcherConfig_screenshot](IMG/check_MAC_linux.png)
+2. Przypisz swojemu urządzeniu statyczny _adres IP_ na routerze
+    * Alternatywnie (nie zawsze działa) ustaw statyczny _adres IP_ dla swojego urządzenia w systemie
+3. Używając `ifconfig` sprawdź czy _adres IP_ urządzenia został zmieniony
+    * TAK - przejdź dalej
+    * NIE - zrestartuj urządzenia
+
+## 2. Udostępnianie folderu do sieci lokalnej
+1. Utwórz folder, który chcesz udostępnić:
+`mkdir /home/<user_name/<path>/<directory_name>`
```
Comment (piotrekvs) [inline Create_local_network_drive/create_local_network_drive.md]: user_name>

File: Create_local_network_drive/create_local_network_drive.md [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+2. Utwórz hasło dla swojego użytkownika dysku sieciowego:
+`sudo smbpasswd -a <user_name>`
+3. Skonfiguruj plik `/etc/samba/smb.conf`:
+    * Na wypadek popełnienia błędu możesz zrobić kopię zapasową pliku _smb.conf_ (`sudo cp /etc/samba/smb.conf /<path_to_backup_dir>`)
+    * `sudo nano /etc/samba/smb.conf` (for braves: można skorzystać z Vim'a zamiast nano tylko trzeba umieć później wyjść)
```
Comment (piotrekvs) [inline Create_local_network_drive/create_local_network_drive.md]: XD

**Comment by mrrys00** [comment]
Merged to master, Feb 14th, 2021

**Pull request merged**